### PR TITLE
[CFM-1674] Set 'c4m_og_content_highlight' flag as global and align it's flagging.

### PR DESCRIPTION
**Pull request by anvmn**
https://issuetracker.amplexor.com/jira/browse/CFM-1674

### CODE REVIEW

File: project/profiles/capacity4more/capacity4more.install [hunk 1]
```diff
@@ -562,3 +562,50 @@ function _set_eu_cookie_compliance() {
 function capacity4more_update_7001() {
   c4m_og_update_groups_contributions();
 }
+
+/**
+ * Sets 'c4m_og_content_highlight' flag as global and aligns it's flagging.
+ */
+function capacity4more_update_7002() {
```
Comment (soniCaH) [inline project/profiles/capacity4more/capacity4more.install]: @anvmn can you please move the update hook to the module defining the flag?

Comment (anvmn) [inline project/profiles/capacity4more/capacity4more.install]: Sure

**Comment by anvmn** [comment]
@nader77 
This is ready for a review...

**Comment by nader77** [comment]
@anvmn Please add a screenshot of the process, just to make sure everything is working as expected.

**Comment by anvmn** [comment]
We start when promote flag is not defined as global:
selection_766

We have a group with 2 nodes promoted:
 - First node 60, and then node 62.
We have a group with 2 nodes promoted:
 - First node 150, and then node 124.
selection_767

After delivery (`drush updatedb -y`):

Flag is set as global:
selection_768

Each group got single node flagged (the last node that was published):
selection_769

**Comment by anvmn** [comment]
@nader77  Done

**Pull request merged**